src/Host.cpp

```diff
@@ -419,9 +419,16 @@ Host::Host(int port, const QString& hostname, const QString& login, const QStrin
     }
     mErrorLogFile.setFileName(logFileName);
     mErrorLogFile.open(QIODevice::Append);
-    // This is NOW used (for map
-    // file auditing and other issues)
+    // This is NOW used (for map file auditing and other issues)
```

Comment:
Not so 'now' any more, this comment got stale. I don't think it adds anything anymore!

Reply:
```suggestion
```